test/periphery/PixCashierUpgradeable.test.ts

```diff
@@ -50,25 +53,30 @@ describe("Contract 'PixCashierUpgradeable'", async () => {
 
     it("Is reverted if caller is not whitelisted", async () => {
       await proveTx(pixCashier.setWhitelistEnabled(true));
-      await expect(pixCashier.cashIn(user.address, tokenAmount))
+      await expect(pixCashier.cashIn(user.address, tokenAmount, TRANSACTION_ID))
         .to.be.revertedWith(REVERT_MESSAGE_IF_ACCOUNT_IS_NOT_WHITELISTED);
     });
 
     it("Is reverted if the contract is paused", async () => {
       await proveTx(pixCashier.setPauser(deployer.address));
       await proveTx(pixCashier.pause());
-      await expect(pixCashier.connect(user).cashIn(cashierClient.address, tokenAmount))
+      await expect(pixCashier.connect(user).cashIn(cashierClient.address, tokenAmount, TRANSACTION_ID))
         .to.be.revertedWith(REVERT_MESSAGE_IF_CONTRACT_IS_PAUSED);
     });
 
     it("Is reverted if the account address is zero", async () => {
-      await expect(pixCashier.connect(user).cashIn(ethers.constants.AddressZero, tokenAmount))
+      await expect(pixCashier.connect(user).cashIn(ethers.constants.AddressZero, tokenAmount, TRANSACTION_ID))
         .to.be.revertedWith(REVERT_MESSAGE_IF_ADDRESS_IS_ZERO);
     });
 
+    it("Is reverted if the transaction id is not provided", async () => {
+      await expect(pixCashier.connect(user).cashIn(ethers.constants.AddressZero, tokenAmount, ethers.constants.HashZero))
+        .to.be.revertedWith(REVERT_MESSAGE_IF_TRANSACTION_ID_IS_NOT_PROVIDED);
+    });
+
     it("Mints correct amount of tokens", async () => {
       await expect(async () => {
-        await proveTx(pixCashier.connect(user).cashIn(cashierClient.address, tokenAmount));
+        await proveTx(pixCashier.connect(user).cashIn(cashierClient.address, tokenAmount, TRANSACTION_ID));
```

Comment:
Codacy has a fix for the issue: [Replace `pixCashier.connect(user).cashIn(cashierClient.address,·tokenAmount,·TRANSACTION_ID)` with `⏎··········pixCashier⏎············.connect(user)⏎············.cashIn(cashierClient.address,·tokenAmount,·TRANSACTION_ID)⏎········`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9795032)
```suggestion
        await proveTx(
```